go/libkb/chain_link.go

```diff
@@ -817,13 +821,16 @@ func (c *ChainLink) Unpack(m MetaContext, trusted bool, selfUID keybase1.UID, pa
 	}
 
 	var payload []byte
-	if trusted && tmp.sigVersion == KeybaseSignatureV1 {
```

Comment:
The `trusted` check is being dropped here.  Is that ok?

Reply:
He!

Reply:
?

Reply:
sorry, i was on the train and lost signal.

i mean "yes"